### ISSUE TITLE
Toggle width does not adjust to longer labels

### ISSUE THREAD

**Issue by bertysentry** [ziscloud/angular-bootstrap-toggle#33]
When using the _on_ and _off_ attributes to set labels with relatively long text (10 to 20 chars), the width of the toggle wrapper doesn't adjust properly.

Example:
```html
<toggle ng-model="$ctrl.someValue" on="My Long Toggle" off="Is Not Well Adjusted"></toggle>
```

This is because the width of the `toggleWrapper` div is based on _offsetWidth_ of the on and off labels, which doesn't include the part of the label that may be clipped and not visible, which is the case with labels in toggle.

Suggestion: Use the _scrollWidth_ property instead

Also: add 24px to adjust for the padding...

**Comment by newroseji** [comment]
Do you have a specific example of the solution?

**Comment by bertysentry** [comment]
I submitted Pull Requests a few days ago with the recommended changes:
https://github.com/ziscloud/angular-bootstrap-toggle/pull/34/files

Note: only 12 pixels need to be added, and not 24 as initially suggested...

**Comment by bertysentry** [comment]
Fixed in #34 and now merged
